source/dstats/distrib.d

```diff
@@ -874,17 +874,28 @@ double chiSquareCDFR(double x, double v) {
  * v = Degrees of freedom. Must be positive.
  *
  */
-double invChiSquareCDFR(double v, double p) {
+deprecated {
+    double invChiSquareCDFR(double v, double p) {
+        return invChiSquareCDFR_Alt(p, v);
+    }
+}
+
+/**
+ * Alternate version of invChiSquareCDFR. After deprecation of current
+ * invChiSquareCDFR, create new invChiSquareCDFR and deprecate an alias
+ * to this function.
+*/
+double invChiSquareCDFR_Alt(double p, double v) {
```

Comment:
need better names than `_Alt`. I'm concerned about anything where two `double` parameters get swapped around, so I'd be happier to create a new function name that we actually keep.

Reply:
ping @jmh530 ?